airmozilla/new/templates/new/_email_new_event.html

```diff
@@ -23,6 +23,19 @@
   Video privacy: <b>{{ event.get_privacy_display() }}</b>
 </p>
 
+<p> 
+  Channels:
+  {% if event.channels.all().count() > 0 %} 
+    <ul>
+    {% for channel in event.channels.all() %}
+      <li>{{ channel | safe}}</li>
```

Comment:
And it's not safe. What if the channel is called `The</li>list` or `Some<script>alert(/xss/)</script>Channel`.